### PR TITLE
Feature/validation: A validation script for assessing our classifirers

### LINKED ISSUE #19: Validate classifiers
Estimate classifiers performance on development and train set using various metrics. 
This can be done e.g. using a jupyter notebook. Results can be presented in readme. 
Suggestions about how it should be done or presented are very welcome

### PR DESCRIPTION
**Pull request by marksverdhei**
Added a script for benchmarking the models on the dev and test set. We need this in order to have some proof on our model performance. Added sklearn dependency

Closes #19

### CODE REVIEW

**Review by LBlend — COMMENTED** [review]
Apparently commit 6c0a05d4b564c3542cce4745ffa04ce21ff35c65 and dab4698b14f7140e98b92bb1b710b70bf8b3568a are causing issues with building the backend. Therefore I will revert these changes for now and merge.

**Comment by LBlend** [comment]
Commits have been reverted (see 551e0df8cf38e6f442d54c1231b16b625c6b403a & c6519af0f9e1f73b8100f89f6c4961bfd5c800da) but doesn't show up here because I'm stupid.

Anyways, LGTM
